site/en/blog/lighthouse-8-4/index.md

```diff
@@ -0,0 +1,54 @@
+---
+layout: 'layouts/blog-post.njk'
+title: What's new in Lighthouse 8.4
+authors:
+  - brendankenny
+date: 2021-09-09
+description: >
+  Lighthouse 8.4 now guards against lazily-loading LCP images and failing FID without a mobile viewport.
+tags:
+  - new-in-lighthouse
+  - lighthouse
+  # - chrome-95
```

Comment:
this Lighthouse release will be in Chrome 95, but adding this tag caused an error (`Could not find i18n result for i18n.common.release_canary_asan`). I assume Chrome releases need to be manually added somewhere?

Reply:
hm I'm surprised to hear that... that seems like a @samthor question

Reply:
re m95: asterisk; https://chromiumdash.appspot.com/commits?commit=649a979e4de2cf38430e46e7198b11ba8a830388&platform=Mac

havent gotten the devtools->chrome roll yet.

Reply:
First, I'm happy for y'all to submit this and I can fix this up later, including uncommenting this line.

In the /tags/ page, we label Chrome releases so you can find articles based on the current stable/beta/etc. Since 95 is currently labeled as Canary, we're looking for that string, but I didn't add it. This isn't super important (/tags/ isn't uh, well used right now) but it's something I want to grow, so folks can find content for specific versions.

Reply:
> In the /tags/ page, we label Chrome releases so you can find articles based on the current stable/beta/etc. Since 95 is currently labeled as Canary, we're looking for that string, but I didn't add it. This isn't super important (/tags/ isn't uh, well used right now) but it's something I want to grow, so folks can find content for specific versions.

SG, thanks @samthor.

FWIW, the new version of Lighthouse always lands in whatever is currently Canary, so this will keep being an issue :) Agreed it's not super important right now, but it will be nice to eventually click the `Chrome-95` tag and easily see the Lighthouse changes that landed in that version.